File: android/app/src/main/java/tech/voidmx/chupapp/MultiphoneMenu.java
package tech.voidmx.chupapp;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;

public class MultiphoneMenu extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_multiphonemenu);

        Button btnCreate, btnJoin;

        btnCreate = findViewById(R.id.btn_createMatch);
        btnJoin = findViewById(R.id.btn_joinMatch);

        btnCreate.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intentGameSets = new Intent(MultiphoneMenu.this,GameSets.class);
                startActivity(intentGameSets);
            }
        });

        btnJoin.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intentJoin = new Intent(MultiphoneMenu.this,JoinMatch.class);
                startActivity(intentJoin);
            }
        });
    }
}

File: android/app/src/main/java/tech/voidmx/chupapp/MultiphoneMenu.kt
package tech.voidmx.chupapp

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.Toast

class MultiphoneMenu : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_multiphone_menu)

        val btCreateMatch = findViewById<Button>(R.id.btCreateMatch)
        val btJoinMatch = findViewById<Button>(R.id.btJoinMatch)

        btCreateMatch.setOnClickListener{
            Toast.makeText(this, "Crear partida", Toast.LENGTH_SHORT).show()
            val intent = Intent(this, GameSets::class.java)
            startActivity(intent)
        }

        btJoinMatch.setOnClickListener{
            Toast.makeText(this, "Unirse a partida", Toast.LENGTH_SHORT).show()
            val intent = Intent(this, JoinMatch::class.java)
            startActivity(intent)
        }
    }
}
